moment/moment-node.d.ts

```diff
@@ -555,6 +557,17 @@ declare namespace moment {
         lts?: string;
     }
 
+    interface MomentParsingFlagsObject {
```

Comment:
minor: you can omit the `Object` part of this interface name, it's a little redundant